packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -715,7 +715,6 @@ export class GarbageCollector implements IGarbageCollector {
 			const containerGCMessage: ContainerRuntimeGCMessage = {
```

Comment:
Just noticed the 3-line comment above this line - should be deleted too.

Reply:
done